src/components/screens/wallet/delegatePerformance/delegatePerformance.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import Box from '@toolbox/box';
+import BoxHeader from '@toolbox/box/header';
+import BoxContent from '@toolbox/box/content';
+import NotFound from '@shared/notFound';
+import { isEmpty } from '@utils/helpers';
+import delegatePerformanceDetails from './delegatePerformanceDetails';
+import styles from './styles.css';
+
+// eslint-disable-next-line max-statements
+const DelegatePerformance = ({ delegate: { error, isLoading, data } } = {}) => {
+  if (!error && isEmpty(data)) {
+    return <div />;
+  }
+
+  if (error && isEmpty(data)) {
+    return <NotFound />;
+  }
+
+  const { dpos: { delegate: { pomHeights, status, consecutiveMissedBlocks } } } = data;
+  let title = '';
+  switch (status) {
+    case 'punished':
+      title = 'Punishment details';
+      break;
+
+    case 'banned':
+      title = 'Banning details';
+      break;
+
+    default:
+      title = 'Unknown details';
```

Comment:
Maybe better doing and object with these properties so then we set tittle as this: `const title = headerTitle[status]`

```
const headerTitle = {
'punished': '...',
...
}```

Reply:
Done